modules/nf-core/rtgtools/svdecompose/main.nf

```diff
@@ -0,0 +1,57 @@
+process RTGTOOLS_SVDECOMPOSE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/rtg-tools:3.12.1--hdfd78af_0':
+        'biocontainers/rtg-tools:3.12.1--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(input), path(tbi)
+
+    output:
+    tuple val(meta), path("*.vcf.gz")    , emit: vcf
+    tuple val(meta), path("*.vcf.gz.tbi"), emit: index
+    path "versions.yml"                  , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args      = task.ext.args ?: ""
+    def prefix    = task.ext.prefix ?: "${meta.id}"
+    def index_vcf = tbi ? "" : "rtg index ${input}"
+    def avail_mem = task.memory.toGiga() + "G"
+    if ("$input" == "${prefix}.vcf.gz") error "Input and output names are the same, set prefix in module configuration to disambiguate!"
+
+    """
+    ${index_vcf}
+
+    rtg RTG_MEM=$avail_mem svdecompose \\
```

Comment:
I dont know the tool just want to make sure if that is the correct usage because it looks like you assign a shell variable